src/structure.rs

```diff
@@ -592,6 +592,13 @@ where
         }
     }
 
+    /// Return the angle, normalized to the range `[-turn_div_2, turn_div_2)`.
+    #[inline]
+    fn normalize_zero(self) -> Self {
```

Comment:
everything about this PR looks good to me, but I'm not sure about the name. Would you find `normalize_signed` to be more descriptive?

Reply:
`normalize_to_turn_div_2()` maybe ?

Reply:
Yeah, I'm not the biggest fan of the name as-is either. I personally like `normalize_signed` more of those options. I'll rebase once there's consensus on a name.

Reply:
`normalize_angle` ?

Reply:
`normalize_for_diff` or `normalize_for_offset` (possibly without `for_` on each).

Reply:
I think `normalize_signed` is most appropriate. @Rhuagh are you strongly opposed?

Reply:
No, I'm just brainstorming

Reply:
Renamed to `normalize_signed`.